requirements-dev.in

```diff
@@ -5,7 +5,7 @@
 ## for pkg in ['hypothesis', 'pytest']:
 ##    print(re.search(f"\n({pkg} *[=>~!].*)\n", versions)[1])
 ##]]]
-hypothesis>=6.0.0
```

Comment:
Whenever any requirement is modified, the tox workflow updating all config files need to be executed as explained here: https://github.com/GridTools/gt4py/blob/main/docs/development/tools/requirements.md